### PR TITLE
Add USDA staking section in Stake

### PR DESCRIPTION
**Pull request by Macxim**
Screen Shot 2022-05-05 at 5 16 37 PM

### CODE REVIEW

File: web/components/stake.tsx [hunk 1]
```diff
@@ -472,6 +486,15 @@ export const Stake = () => {
       setLpStxDikoPendingRewards(stxDikoLpPendingRewards);
       setLpStxXbtcPendingRewards(stxXbtcLpPendingRewards);
       setLpXbtcUsdaPendingRewards(xbtcUsdaLpPendingRewards);
+
+      // USDA Staking in Liquidation Pool
+      setTotalPooledUsda(totalPooledUsda);
+      setUserPooledUsda(userPooledUsda);
+      setDikoRewardsToAdd(dikoEpochRewardsToAdd);
```
Comment (philipdesmedt) [inline web/components/stake.tsx:493]: REQ: this is never used, so remove this

**Review by philipdesmedt — CHANGES_REQUESTED** [review]

File: web/components/stake.tsx (hunk 1 above)
Comment (Macxim) [reply web/components/stake.tsx:493]: It's used on L495 though.

File: web/components/stake.tsx [hunk 2]
```diff
@@ -490,8 +489,8 @@ export const Stake = () => {
       // USDA Staking in Liquidation Pool
       setTotalPooledUsda(totalPooledUsda);
       setUserPooledUsda(userPooledUsda);
-      setDikoRewardsToAdd(dikoEpochRewardsToAdd);
 
+      console.log(dikoEpochRewardsToAdd)
```
Comment (philipdesmedt) [inline web/components/stake.tsx:493]: @Macxim pls remove this console log :D

Comment (Macxim) [reply web/components/stake.tsx]: 😅

**Pull request merged**